lib/init.coffee

```diff
@@ -11,4 +20,39 @@ module.exports =
         'Available codes: https://pypi.python.org/pypi/pep257#error-codes')
 
   activate: ->
-    console.log 'activate linter-pep257'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-pep257.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @subscriptions.add atom.config.observe 'linter-pep257.ignoreCodes',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+
+  provideLinter: ->
+    provider =
+      grammarScopes: ['source.python']
+      scope: 'file' # or 'project'
+      lintOnFly: true # must be false for scope: 'project'
+      lint: (textEditor)->
+        filePath = textEditor.getPath()
+        type = 'Warning'    # all messages are of type "Warning"
+        cmd = atom.config.get('linter-pep257.executablePath')
+        params = [filePath]
+        if @executablePath is ''
```

Comment:
It is never ``, because when it's empty, atom sets it's value to the default value